src/app.rs

```diff
@@ -50,11 +53,7 @@ impl App {
     /// on the tree state will also error.
     #[instrument(name = "App::new", level = "debug", skip_all)]
     pub async fn new(config: Config) -> anyhow::Result<Arc<Self>> {
-        let ethereum = Ethereum::new(&config);
-        let db = Database::new(&config.database);
-
-        let (ethereum, db) = tokio::try_join!(ethereum, db)?;
```

Comment:
I have removed that sync on etherum and database connnecting. Let me know if this was important part to speed up initialization as I guess connection should be quite fast.

Reply:
Unless there's a noticable difference in tests that's fine by me